auto-run-nondex/runNondexAll.sh

```diff
@@ -0,0 +1,5 @@
+# Assume this script is run from the folder of .pom.xml files
+
+mvn edu.illinois:nondex-maven-plugin:2.1.1-SNAPSHOT:nondex -Dlicense.skip=true -Drat.skip=true -fae | tee ./nondex.log
```

Comment:
You don't want `2.1.1-SNAPSHOT` (because 2.1.1 has been released), and you don't need `=true`. It'd be hard to review all other changes, which are rather big. Can you first propose a new PR with the minimal changes for the functionality you want to add?